core/themes/basis/css/component/small-text-components.css

```diff
@@ -176,22 +176,15 @@ ul.links,
 }
 
 .links.inline li:after {
-  content: ',';
+  content: '|';
   position: static;
   display: inline;
   -webkit-transform: none;
   transform: none;
   border: 0;
+  padding: 0 0 0 0.6em;
```

Comment:
This looks like it would need a matching RTL rule (padding is on left side only here)